test/form/samples/output-named-library/_expected/system.js

```diff
@@ -0,0 +1,10 @@
+System.register('libraryName', [], function (exports, module) {
```

Comment:
Before my pr, `'libraryName', ` would not have appeared in this System.register call